libs/driver/shopify/src/graphql/create-apollo-config.ts

```diff
@@ -0,0 +1,38 @@
+import { inject } from '@angular/core';
+import { InMemoryCache } from '@apollo/client/cache';
+import { ApolloLink } from '@apollo/client/core';
+import { setContext } from '@apollo/client/link/context';
+import { onError } from '@apollo/client/link/error';
+import { HttpLink } from 'apollo-angular/http';
+
+export function createApolloConfig({
```

Comment:
This API is unclear, what is `uri`? and what are `headers`? Are users expected to know how to configure this?

The API of `shopifyDriverConfig` is more user-friendly and I think captures the true intent of this function.